### DISCUSSION TITLE
web layout

### DISCUSSION THREAD

**Discussion by gertcuykens** [fastapi/fastapi#5488] [Questions] [answered]
https://fastapi.tiangolo.com/

<img width="396" alt="image" src="https://user-images.githubusercontent.com/863731/195380490-dcabb2ed-8bd6-4f9f-ab56-d8fd5985c239.png">

I think web layout can be reorganised so there is less clutter by removing 'fastapi' headers and just language selector in the title bar so you end up more in line like this layout

<img width="881" alt="image" src="https://user-images.githubusercontent.com/863731/195382135-bf4b4e91-ff63-46f0-9685-556c9f418db5.png">

**Comment by tiangolo** [comment, marked as answer]
So, your suggestion is renaming the first thing called "FastAPI" to "Introduction"?

**Reply by gertcuykens** [reply]
yes and removing the text fast api above logo and also languages from site menu because its already available in top
<img width="678" alt="image" src="https://user-images.githubusercontent.com/863731/201380491-c0b1aff9-f27a-4881-ad59-c4e4e25133e6.png">

**Reply by tiangolo** [reply]
Thanks for the suggestion. About the language, I'll keep it there. About the text, but sure if that would affect search engines that wouldn't find the title of that page.

Anyway, maybe I'll revisit that later, for now I have to focus on a lot of other things first. But thanks for the interest!

**Reply by gertcuykens** [reply]
Maybe cut the circle of the image and recreate the logo [x] FastAPI part image part text
